portfolio/src/main/webapp/script.js

```diff
@@ -47,11 +47,44 @@ document.addEventListener('DOMContentLoaded', function() {
       navbar.classList.remove('sticky');
     }
   }
+})
+/**
+ * Fetches comments from the server and adds them to the DOM.
+ */
+function loadComments() {
+  fetch('/comment').then(response => response.json()).then((comments) => {
+    const commentListElement = document.getElementById('comment-list');
+    comments.forEach((comment) => {
+      commentListElement.appendChild(createCommentElement(comment));
+    })
+  });
 }
 
-function getMessage() {
-  fetch('/comment').then(response => response.text()).then((quote) => {
-    document.getElementById('quote-container').innerText = quote;
-    console.log(quote);
+function createCommentElement(comment) {
+  const commentElement = document.createElement('li');
+  commentElement.className = 'comment';
+
+  const titleElement = document.createElement('span');
+  titleElement.innerText = comment.content;
+  console.log(comment.content);
+
+  const deleteButtonElement = document.createElement('button');
+  deleteButtonElement.innerText = 'Delete';
+  deleteButtonElement.addEventListener('click', () => {
+    deleteComment(comment);
+
+    // Remove the task from the DOM.
```

Comment:
Nit: "Remove the _comment_ from the DOM."

Reply:
Done.